src/palace/manager/sqlalchemy/model/patron.py

```diff
@@ -684,13 +725,15 @@ class Annotation(Base):
     target = Column(Unicode)
 
     @classmethod
-    def get_one_or_create(self, _db, patron, *args, **kwargs):
+    def get_one_or_create(
```

Comment:
Honestly I don't see the value of that function at all. The comment says:
> find or create an Annotation, but only if the patron has annotation sync turned on

But it clearly doesn't check if annotation sync is turned on 😟. I'll just 🔥 it, since its misleading.